src/screens/authentication/create_new_wallet/GetPublicKeyScreen/index.tsx

```diff
@@ -63,19 +65,50 @@ const GetPublicKeyScreen = ({ transport, setTransport }: Props) => {
       if (ledgerPublicKey) {
```

Comment:
Please move `setIsLoading(true);` in line 60 to line 65 (above `if(ledgerPublicKey)`).
Test case: click Connect Ledger, turn on Ledger but not open Casper App in Ledger. Click the ledger on the app screen.

https://user-images.githubusercontent.com/34257375/157459856-0d69b85a-eedc-4132-9e3a-fa85766be155.MP4

Reply:
Nice catch. I fixed it in https://github.com/CasperDash/casperdash-mobile-wallet/pull/52/commits/e98f8b7ca8a9674c7e38ddba630c0bb54f977aba